lms/djangoapps/certificates/views/webview.py

```diff
@@ -236,7 +237,7 @@ def _update_course_context(request, context, course, course_key, platform_name):
     context['full_course_image_url'] = request.build_absolute_uri(course_image_url(course))
     course_title_from_cert = context['certificate_data'].get('course_title', '')
     accomplishment_copy_course_name = course_title_from_cert if course_title_from_cert else course.display_name
-    context['accomplishment_copy_course_name'] = accomplishment_copy_course_name
+    context['accomplishment_copy_course_name'] = bleach.clean(accomplishment_copy_course_name, tags=['br'], strip=True)
```

Comment:
I'd move this to a Mako filter and use it in the template.  You could keep it as two PRs if you want to add the filter as a separate PR.  See [my comment](https://github.com/edx/web-certificates/pull/55#discussion_r156088103) in the other PR for details.